weekly-updates/Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -18,6 +19,89 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
 
 ---
 
+## Date: 7 March
+
+*Updates covering 2 weeks as I had a few days off at the start of March.*
+
+### What brought you joy this (2) weeks?
+
+* LA helped me SO SO much going through the knowledge share slide by slide and brainstorming the take home points for my presentation 🙏 😻
+* Great call with EM about systems design.
+* Helped a friend with a job presentation (loved it, but also - boo free work!!)
+* Super helpful call with JA getting some guidance on a project.
+* Super helpful call with CF on TPS finances 🙏
+* TPS seminar - so appreciate the eScience Center's team contributing to The Turing Way 💗.
+* I gave some - hopefully - useful context to someone applying to join the Turing as a (equivalent to a) research fellow.
+* BIDS Steering Group discussion about adding species to the standard was amazing.
+* All the lab mentorship notes on GitHub are bringing me so much joy 🙌
+* GA & JS gave a great talk at the Social Data Science special interest group!
+  And they were really well supported by lab members.
+* Really appreciated the conversion with the Exec Director ladies.
+* Super helpful mentorship call with KP 🌱
+* Really really no meeting Friday got through a lot of emails!
+  5513 --> 5392 (but then back up again by the time I got back from my dog walk 😂)
+  * https://twitter.com/kirstie_j/status/1367764683864875009?s=20
+* AB got some finances sorted out which I _hope_ will make one of our TPS project leads feel a bit better 🌼
+* I finished reading Thundehead and absolutely loved it.
+
+### What did you achieve
+
+* Presentation to the DECOVID Exec team.
+  Went well.
+  PHEW 😌
+* All hands meeting was helpful too.
+* Recruited 2 students to work on a project 🚀 🌟
+* Wrote letter of recommendation for PR.
+  Fingers crossed 🤞😬🤞
+* Got an ethics approval over the finish line.
+* Gave a report writing workshop for the DSG participants 📝
+* Research Fish submission.
+  * And the correction 🙈
+* I was helpful on a call with GT
+* I think I was helpful on a call with VF
+* Useful catch up chat with AM
+* Met the new fast streamer for TPS 👋
+* Brainstorming meeting for training at the Turing
+* Catch up with MBr
+* Bit of brainstorming on Trinity Challenge... but not really enough...
+* Bit of brainstorming for future resourcing for The Turing Way
+* 1:1 w AB, MS, EK, GA & ME
+* Reviewed the DECOVID newsletter!
+  Loved it!
+* Wrote huge complain about how the Turing slack is under utilised 😬
+* BIDS Steering Group meeting
+* EDoN meeting
+* I had 3 days off! 🙀
+  They were really great 🌟
+
+### What did you struggle with?
+
+* I should have helped someone more than I did.
+  It was a tough run because I also needed to fit my own oxygen mask.
+  But I'm sad at how things turned out.
+* EAG work was harder than it should have been.
+* I'm disappointed with engagement across the Turing in general.
+  I don't know what is up but there's something about 2021 that has just taken all the wind out of everyone's sails.
```

Comment:
I think it's just something that ebbs and flows quite a bit. At least that's what I've experienced. And I think the mentoring sessions have been a wonderful addition to the type of support provided.

Reply:
I agree with both of you: there is low engagement and general mental blerugh at the moment but I also think it will naturally ebb & flow (at least I hope it starts flowing a bit more!) 

Also +1 to mentoring sessions being super useful